docs/framework/react/api/router/RouterType.md

```diff
@@ -112,6 +113,12 @@ Commits a new location object to the browser history.
     - Optional
     - Defaults to `true` so that the scroll position will be reset to 0,0 after the location is committed to the browser history.
     - If `false`, the scroll position will not be reset to 0,0 after the location is committed to history.
+  - `hashChangeScrollIntoView`
```

Comment:
we should also add a `defaultHashChangeScrollIntoView` (or whatever this thing will be named finally) into router (defaults to `true`)

Reply:
I added it to the router options and implemented it as I saw some of the other default options. Thanks!